components/ui-api-layer/internal/domain/remoteenvironment/re_service_test.go

```diff
@@ -243,6 +249,138 @@ func TestGetConnectionURLFailure(t *testing.T) {
 	})
 }
 
+func TestRemoteEnvironmentService_Create(t *testing.T) {
+	// GIVEN
+	client := fake.NewSimpleClientset()
+	fixName := "fix-name"
+	fixDesc := "desc"
+	labels := map[string]interface{}{
+		"fix": "lab",
+	}
+	expectedLabels := map[string]string{
+		"fix": "lab",
+	}
+
+	svc, err := remoteenvironment.NewRemoteEnvironmentService(client.ApplicationconnectorV1alpha1(), remoteenvironment.Config{}, newDummyInformer(), nil, nil)
+	require.NoError(t, err)
+
+	// WHEN
+	re, err := svc.Create(fixName, fixDesc, labels)
+	require.NoError(t, err)
```

Comment:
`require.NoError(t, err)` statement should be under `// THEN` section

same for all other occurrences